erizo_controller/erizoClient/src/webrtc-stacks/BaseStack.js

```diff
@@ -16,6 +16,22 @@ const BaseStack = (specInput) => {
   let isNegotiating = false;
   let latestSessionVersion = -1;
 
+  const parseMediaConfiguration = () => {
+    const codecs = { audio: null, video: null };
+    switch (specBase.mediaConfiguration) {
+      case 'default':
+        codecs.video = 'VP8';
+        codecs.audio = 'OPUS';
+        break;
+      default:
+        codecs.video = specBase.mediaConfiguration.split('_AND_')[0];
```

Comment:
Idk if the controller actually knows the mediaconfiguration of the room, so I choose to pass it inside the token since it's something it's stored in nuve

Reply:
the controller does not know it currently, but it could be changed to support p2p cases too. The issue that I see with this approach is that we're using the mediaconfiguration name, which does not even follow a convention, and you're parsing the name by splitting `_AND_`. 
I'd say that for having a consisten solution across p2p and not p2p modes, we should do something similcast to what we have in `ErizoJS/models/Connection._getMediaConfiguration`: return the complete media configuration JSON to the client and filter the SDP with that info instead of the substring VP8/OPUS/etc. Otherwise we would have different solutions for p2p and not p2p cases.